item.go

```diff
@@ -18,7 +20,7 @@ type Item struct {
 	ID        string `json:"Id,omitempty"`
 	SyncToken string `json:",omitempty"`
 	//MetaData
-	Name        string
+	Name        string `json:"Name"`
```

Comment:
You should probably add the `omitempty` value to the struct tag. It looks like these are optional fields, or they presumably would have already been here?